AGPs/AGP-20.md

```diff
@@ -0,0 +1,23 @@
+---
+AGP: 20
+Title: Keep Aragon Focused on Ethereum, not Polkadot
+Author: Ameen Soleimani (@ameensol)
+Status: Draft
+Track: Finance
+Created: 4-11-2019
+---
+
+# AGP-20: Keep Aragon Focused on Ethereum, not Polkadot
+
+**Version 0.1**
+
+### Proposal
+
+Much of the support that Aragon received from the Ethereum community, both financially and otherwise, was because Aragon was building *on Ethereum*. Today it is becoming more clear that Aragon intends to build "AragonChain" as a parachain Polkadot, an Ethereum competitor. As many ANT holders are significant stakeholders in the Ethereum ecosystem, this quite possibly goes against their wishes. Those of us who wish to see Aragon remain focused on building on Ethereum, unless consent is granted to build on Polkadot. After all, it is the ANT holders whose money is being used to fund Aragon's ongoing development.
```

Comment:
```suggestion
Much of the support that Aragon received from the Ethereum community, both financially and otherwise, was because Aragon was building *on Ethereum*. Today it is becoming more clear that Aragon intends to build "AragonChain" as a parachain on Polkadot, an Ethereum competitor. As many ANT holders are significant stakeholders in the Ethereum ecosystem, this quite possibly goes against their wishes. There are those of us who wish to see Aragon remain focused on building on Ethereum, unless consent is granted to build on Polkadot. After all, it is the ANT holders whose money is being used to fund Aragon's ongoing development.
```